plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,57 +6,51 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, states }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
+export default function Breadcrumbs({ jobPath, jobName, jobInfo, children }) {
+  let breadcrumbParts = [
+    <Breadcrumb key={"Jobs"} title="Jobs">
+      <BreadcrumbTextContent>
+        <Link to={"/jobs/overview/"}>{"Jobs"}</Link>
+      </BreadcrumbTextContent>
+    </Breadcrumb>
+  ];
+
+  if (jobPath !== undefined) {
+    const pathParts = jobPath.map((jobPathPart, index) => {
+      const partId = jobPath.slice(0, index + 1).join(".");
+
+      return (
+        <Breadcrumb key={jobPathPart} title="Jobs">
+          <BreadcrumbTextContent>
+            <Link to={"/jobs/overview/" + partId}>{jobPathPart}</Link>
+          </BreadcrumbTextContent>
+        </Breadcrumb>
+      );
+    });
+    breadcrumbParts = breadcrumbParts.concat(pathParts);
+  }
 
-    return (
-      <Breadcrumb key={id} title="Jobs">
+  if (jobName !== undefined) {
+    const partId = jobPath.concat([jobName]).join(".");
+    breadcrumbParts = breadcrumbParts.concat([
+      <Breadcrumb key={jobName} title="Jobs">
         <BreadcrumbTextContent>
-          <Link to={link}>{name === "" ? "Jobs" : name}</Link>
+          <Link to={"/jobs/detail/" + partId}>{jobName}</Link>
         </BreadcrumbTextContent>
-
-        {isDetailPage ? states : null}
+        {jobInfo}
```

Comment:
If we could have `jobInfo` not as a prop but as a component that could become 

```js
<JobInfo job={job} />
```

Reply:
no, that is exactly what we just got rid of… 😭 (see comment above)